### PR TITLE
ZPS-1006: Apply FileSystem_NFS_Client template to all NFS filesystems.

### PR DESCRIPTION
**Pull request by byshovets**
Fixes [ZPS-1006](https://jira.zenoss.com/browse/ZPS-1006).

The old one approach was broken due to recent ZPL updates.

### CODE REVIEW

File: ZenPacks/zenoss/LinuxMonitor/FileSystem.py [hunk 1]
```diff
@@ -86,19 +86,24 @@ def impacting_object(self):
...
+        For NFS mounts replace general FileSystem template with NFS-specific one.
         """
-        storage_server = list(self.getStorageServers())
-        if storage_server:
-            return "FileSystem_NFS_Client"
-        return "FileSystem"
+        old_templates = super(FileSystem, self).getRRDTemplates()
+
+        if self.type != 'nfs':
+            return old_templates
```
Comment (ssOleg) [inline ZenPacks/zenoss/LinuxMonitor/FileSystem.py]: Does it work for all NFS Filesystems now?

Comment (byshovets) [inline ZenPacks/zenoss/LinuxMonitor/FileSystem.py]: @ssOleg: yes

Comment (byshovets) [inline ZenPacks/zenoss/LinuxMonitor/FileSystem.py]: @ssOleg do you approve this change?

Comment (ssOleg) [inline ZenPacks/zenoss/LinuxMonitor/FileSystem.py]: approved

File: ZenPacks/zenoss/LinuxMonitor/FileSystem.py [hunk 2]
```diff
@@ -86,19 +86,24 @@ def impacting_object(self):
...
+
+        new_templates = []
+
+        for template in old_templates:
+            if 'FileSystem' in template.id and 'NFS_Client' not in template.id:
+                new_templates.append(self.getRRDTemplateByName('FileSystem_NFS_Client'))
```
Comment (cluther) [inline ZenPacks/zenoss/LinuxMonitor/FileSystem.py]: Are you attempting to deal with the possible _-replacement_ and _-addition_ custom templates used by zenpacklib here? If so, something like the following would be more comprehensive.

```python
new_templates.append(
    self.getRRDTemplateByName(
        template.id.replace("FileSystem", "FileSystem_NFS_Client")))
```

**Pull request merged**